spec/classes/dhcp_spec.rb

```diff
@@ -4,6 +4,7 @@
     {
       'dnsdomain'           => ['sampledomain.com', '1.1.1.in-addr.arpa'],
       'nameservers'         => ['1.1.1.1'],
+      'nameservers_ipv6'    => ['1:5ee:bad:c0de'],
```

Comment:
IMHO it's a best practice to input valid data, even in tests. While currently there's no input validation, it's better to be ready for it.